### ISSUE TITLE
Use existing dependency container when running application locally using Tye

### ISSUE THREAD

**Issue by kassemdarwish** [dotnet/tye#939]
This is the first time I try to use Tye so bare with me if I'm going wrong about it :)

I tried to reference an existing container which is running in Docker Desktop in the tye.yml file (as follows)
`- name: postgres
  image: postgres
  bindings: 
    - port: 5432
      connectionString: <redacted>`
But when the application ran, it tried to create another postgres container and it failed:
`[13:20:12 ERR] docker run failed for postgres with exit code 125:`

Is there a way to make Tye use the existing container instead of trying to create a new one? And let's assume that I did not have an existing container and therefore Tye managed to run properly the first time, what happens when I stop the application and run again? Is it going to drop the container and re-create it?

**Comment by jkotalik** [comment]
> Is there a way to make Tye use the existing container instead of trying to create a new one?

No, but we'd like to enable this as an option. You may be able to make this work by using `external` https://github.com/dotnet/tye/blob/b31204486cb10a844bce0b142118a234850574d3/docs/reference/local_development.md#loading-the-application. That will make tye not run the image.

> Is it going to drop the container and re-create it?

Yes, it will stop the container and start it again.
